pom.xml

```diff
@@ -220,6 +220,9 @@
     <netty.version>4.1.109.Final</netty.version>
     <netty-tcnative.version>2.0.65.Final</netty-tcnative.version>
     <icu4j.version>72.1</icu4j.version>
+    <junit-jupiter.version>5.9.1</junit-jupiter.version>
```

Comment:
May I ask why we downgrade this, @pan3793 ? Previously, we have `5.9.3`.

https://github.com/apache/spark/blob/ac9a12ef6e062ae07e878e202521b22de9979a17/pom.xml#L1239

Reply:
```suggestion
    <junit-jupiter.version>5.9.3</junit-jupiter.version>
```

Reply:
@dongjoon-hyun sorry, that's a typo, corrected